### DISCUSSION TITLE
How change preferences which languages or technologies i want to see on my front page

### DISCUSSION THREAD

**Discussion by malisoft** [dailydotdev/daily#1247] [Feature Requests]
i don't use flutter any more, so i want delete this framework from my preferences how can i do that, if exist please can you put it into the sidebar? is so hard to find it

**Comment by malisoft** [comment]
i find it, but can you make please a float button? because scrolling it despairs

**Reply by idoshamun** [reply]
Hi @malisoft,

Currently we don't have plans to make the button floating but we will consider it. Thanks for reaching out!
